sparkly/catalog.py

```diff
@@ -116,17 +117,13 @@ def has_table(self, table_name):
         Returns:
             bool
         """
-        db_name = get_db_name(table_name)
-        rel_table_name = get_table_name(table_name)
 
-        if not self.has_database(db_name):
+        try:
+            self._spark.sql('SELECT 1 FROM {} WHERE 1=0'.format(table_name))
```

Comment:
why do we need where here?

Reply:
hmm true i guess it won't actually _do_ anything... i'm just being paranoid, and it's a reflex from doing this kinda thing in other dbs.

but it does have the convenience of suggesting to the user that we don't actually care about the data, just that the table exists... which if I got rid of the where clause i would probably put a comment in about it.